### PR TITLE
Extends Airlock Sight

### PR DESCRIPTION
**Pull request by Zytolg**
## About The Pull Request

Changes a variable in airlock code to extend airlock vision further, and enabling the airlock helpers on the holodeck to work again. 

## Why It's Good For The Game

I've been doing map cleanup guided by logging output to map_errors.txt. Airlock cyclelink helpers were tossing out errors in the holodeck because they couldn't see far enough to make a connection with their partners. This also meant that those doors weren't cycling as expected. At that point we could either remove the helpers, or fix the problem, and maptainer prefers this solution. AnturK on discord also mentioned that world.view isn't really something that should be used for this.

MappingErrorLog

## Changelog
:cl:
fix: Airlock Cyclelink Helpers can now function on the holodeck again.
code: Airlocks can see out to 11 tiles now
/:cl:

### CODE REVIEW

File: code/game/machinery/doors/airlock.dm [hunk 1]
```diff
@@ -191,7 +191,7 @@
 		cyclelinkedairlock = null
 	if (!cyclelinkeddir)
 		return
-	var/limit = world.view
+	var/limit = 11
```
Comment (Mothblocks) [inline code/game/machinery/doors/airlock.dm:194]: Where does this number come from? Make it a define.

**Comment by Zytolg** [comment]
_Assigns it to a define_, my smooth brain jumbled up the terms when writing that contribution note.

File: code/game/machinery/doors/airlock.dm [hunk 2]
```diff
@@ -57,6 +57,8 @@
 
 #define DOOR_CLOSE_WAIT 60 /// Time before a door closes, if not overridden
 
+#define DOOR_VISION_DISTANCE 11 ///The maximum distance a door will see out to
```
Comment (Mothblocks) [inline code/game/machinery/doors/airlock.dm:60]: #undef it at the end as well.

**Review by Mothblocks — CHANGES_REQUESTED** [review]
Almost there.

**Pull request merged**